app/controllers/observations_controller/new_and_create.rb

```diff
@@ -194,6 +195,26 @@ def validate_place_name(params)
     success
   end
 
+  def validate_project_checkboxes!
+    return true if params[:project].empty? ||
+                   params[:project][:ignore_proj_dates]
+
+    @suspect_checked_projects = checked_out_of_range_projects
+    @suspect_checked_projects.empty?
+  end
+
+  def checked_out_of_range_projects
+    checked_proj_check_boxes =
+      params[:project].select { |_, value| value == "1" }.keys
+    return [] if checked_proj_check_boxes.none?
+
+    checked_proj_ids =
+      checked_proj_check_boxes.map { |str| str.gsub("id_", "") }
+    # Get the AR records so that we can call Project methods on them
+    Project.where(id: checked_proj_ids).
```

Comment:
Question: Should I try to turn Project#dates_exclude? into an ARELized, scoped, AR query (instead of a method)?
Maybe I could avoid the filter and just do something like:
`Project.where(id: checked_proj_ids).where(dates_exclude(@observation.when)`